_posts/2022-09-08-bazel-community-roundup.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: posts
+title: "Bazel Community Round-up for August 2022"
+authors:
+  - jin
+---
+
+Welcome to the August issue of the Bazel Community Round-up, where we collect and share
+content about Bazel over the month.
+
+**Announcements**
+
+*   Call for Proposals for BazelCon 2022 (Nov 16-17, New York City) is now open. [Submit your proposal here.](https://bit.ly/bazelcon2022cfp)
+    *   [BazelCon home page](https://conf.bazel.build/) 
+    *   [bazel-discuss thread](https://groups.google.com/g/bazel-discuss/c/QvNCK8z0ha8)
+*   Slack is now kindly sponsoring https://slack.bazel.build, thanks to Erik Kerber.
```

Comment:
Did you intentionally not linkify the slack link?

Reply:
Not intentional - linkified.